Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Endpoints/EndpointResolverMiddleware.swift

```diff
@@ -18,35 +18,37 @@ import enum ClientRuntime.EndpointsAuthScheme
 import protocol ClientRuntime.EndpointsAuthSchemeResolver
 import protocol ClientRuntime.EndpointsRequestContextProviding
 
-public struct EndpointResolverMiddleware<OperationStackOutput, Params: EndpointsRequestContextProviding> {
+@_spi(AWSEndpointResolverMiddleware)
+public struct AWSEndpointResolverMiddleware<OperationStackOutput, Params: EndpointsRequestContextProviding> {
```

Comment:
This middleware is changed to take a endpoint params block, since account ID cannot be resolved until after auth scheme has been resolved & credentials are sourced (and account ID along with it.)